sdk/javascript/client/deferred.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 Intel Corporation
+ * Copyright 2017 Intel Corporation
```

Comment:
My understanding is that if a file was originally made in 2016, but then updated in 2017 it would be "Copyright 2016, 2017 Intel Corporation"